.helm-production.yml

```diff
@@ -0,0 +1,7 @@
+ingress: 
+  globalStaticIpName: landing-production
+  hosts:
+    - sourced.tech
+    - www.sourced.tech
+nodeSelector:
+  cloud.google.com/gke-nodepool:default-pool
```

Comment:
nit: please tell your editor to add new lines at the end of the last line of a file :)